example/cumulus-tf/browse_example.tf

```diff
@@ -256,7 +256,9 @@ module "cookbook_browse_example_workflow" {
             "buckets": "{$.meta.buckets}",
             "distribution_endpoint": "{$.meta.distribution_endpoint}",
             "collection": "{$.meta.collection}",
-            "duplicateHandling": "{$.meta.collection.duplicateHandling}"
+            "duplicateHandling": "{$.meta.collection.duplicateHandling}",
+            "cmr": "{$.meta.cmr}",
```

Comment:
I don't know how `MoveGranules` _ever_ worked. As far as I can tell, it was never provided CMR credentials.